backends/alpm/pk-alpm-config.c

```diff
@@ -922,8 +946,9 @@ pk_alpm_config_configure_alpm (PkBackend *backend, PkAlpmConfig *config, GError
 	g_return_val_if_fail (config != NULL, FALSE);
 
 	handle = pk_alpm_config_initialize_alpm (config, error);
-	if (handle == NULL)
+	if (handle == NULL) {
 		return NULL;
+	}
```

Comment:
Why is this style change being done? We don't typically do curly braces on one-line conditionals.